project/fixtures.py

```diff
@@ -1,15 +1,22 @@
 import json
 import os
-from pydoc import locate
 from config import FIXTURES_DIR
+from project.database import db_session as session
 
 
-def load_fixtures(session, filepath):
+def import_class(cl):
+    d = cl.rfind(".")
+    classname = cl[d+1:len(cl)]
+    m = __import__(cl[0:d], globals(), locals(), [classname])
```

Comment:
А ты поиграйся с командами в дебаггере. Магии тут особой нет, если понимать, чтo ключевое слово *import* транслируется в вызовы __import__() ([lurk moar](https://docs.python.org/3.4/library/functions.html#__import__))

> Note how __import__() returns the toplevel module here because this is the object that is bound to a name by the import statement.
```python
_temp = __import__('spam.ham', globals(), locals(), ['eggs', 'sausage'], 0)
eggs = _temp.eggs
saus = _temp.sausage
```

То есть, ты вручную импортируешь пакет по его квалифицированному имени (предварительно вытянув из него имя класса) в locals-globals данного контекста.

Но вот вручную писать такие костыли не надо, потому что они уже написаны.
→ [importlib](https://docs.python.org/3.4/library/importlib.html)

> importlib.import_module(name, package=None)

> Import a module. The name argument specifies what module to import in absolute or relative terms (e.g. either pkg.mod or ..mod). If the name is specified in relative terms, then the package argument must be set to the name of the package which is to act as the anchor for resolving the package name (e.g. import_module('..mod', 'pkg.subpkg') will import pkg.mod).